src/helpers/graph.js

```diff
@@ -0,0 +1,42 @@
+export function removeNoteFromGraph(nodes, edges, noteToDelete) {
+  let newNodes = [ ...nodes ]
+  let newEdges = [ ...edges ]
+  let parentEdge = newEdges.find(edge => edge.to === noteToDelete.id)
+
+  removeChild(noteToDelete.id)
+  newNodes = newEdges.find(edge => edge.from === parentEdge.from) ? //check if there are siblings of deleted node if not revoke parant status

```

Comment:
1. No space after `//`.
2. This line is too long (combined with the comment).
3. parAnt -> parEnt ?
4. A comment is a code smell. Can't we declare functions instead of using the comment?
    I.e.
   ```
   function doesDeletedNodeHasSiblings() { ... }
   function revokeParentStatus() { ... }
   ```
5. What does it mean to revoke a parent status? I don't remember the nodes to have any status. Which statuses can nodes have?